### DISCUSSION TITLE
Personal note issues. Doesn't safe

### DISCUSSION THREAD

**Discussion by U255tk** [cgeo/cgeo#16254] [Q&A]
Hey,

I have used C:GEO for more than a year. And recently I started to have problems with the app not saving the notes I make on different caches. 

It's safes them right away after I make the note but after a couple of days the note is gone. 

Can anyone help out?

**Comment by moving-bits** [comment]
Do you store the cache into a list in c:geo?

Caches which are not stored in one or more lists in c:geo are considered "deleted" and will be purged automatically after a grace period of 72h.

**Reply by U255tk** [reply]
Thanks. Okay I can see that helps if I store them to a list. 
But earlier I didn't have to safe them in the stored list for the notes to be saved. Normally after I do the note I delete them from the stored list so they don't mix up with the drafts I have.

**Comment by bekuno** [comment]
If you write a personal note to an unsaved cache then the cache is stored in the list "Stored".

**Reply by U255tk** [reply]
Thanks. Okay I can see that helps. But earlier I didn't have to safe them in the stored list for the notes to be saved. Normally after I do the note I delete them from the stored list so they don't mix up with the drafts I have.

**Reply by moving-bits** [reply]
You can create additional lists, eg. "done", and store the cache there.

Or, if you have a premium membership at geocaching.com, you can upload your personal note to their server.

**Comment by bekuno** [comment]
The local personal note is not automatic uploaded to GC. Maybe you did the manual upload "earlier"?
